recipes/default.rb

```diff
@@ -21,12 +18,12 @@
   end
 end
 
-elasticsearch_configure 'elasticsearch' do
-  node['elasticsearch']['configure'].each do |key, value|
-    # Skip nils, use false if you want to disable something.
-    send(key, value) unless value.nil?
-  end
-end
+# elasticsearch_configure 'elasticsearch' do
```

Comment:
The changelog mentions removing the recipe but it's still partially here, some commented out and some remaining.

Reply:
Ah right OK(!) 

That's not a pattern we support in any of the other cookbooks and I wouldn't suggest it's a good pattern. But we should definitely support it right now 👍🏼

Reply:
Yeah, I didn't love supporting it but it's something from before I inherited working on the cookbook.